### PR TITLE
Replace remote GitHub avatars with local assets

### PR DESCRIPTION
**Pull request by joshfarrant**
## Summary

- Removed remote usage of https://github.com/mona.png and replaced with local asset instead.
- Modified types in `IDE` component to remove unused `avatar` property from assistant message type
  - This allowed me to remove the unused assistant avatars from fixture data. They were set to https://github.com/copilot.png, which doesn't resolve anyway.

## What should reviewers focus on?

- Check I've not missed any instances of remote GitHub avatars
- Check that nothing has changed visually in the docs or Storybook

## Steps to test:

1.
1.
1.

## Supporting resources (related issues, external links, etc):

-
-

## Contributor checklist:

- [x] All new and existing CI checks pass
- [x] Tests prove that the feature works and covers both happy and unhappy paths
- [x] Any drop in coverage, breaking changes or regressions have been documented above
- [x] UI Changes contain new visual snapshots (generated by adding `update snapshots` label to the PR)
- [x] All developer debugging and non-functional logging has been removed
- [x] Related issues have been referenced in the PR description

## Reviewer checklist:

- [ ] Check that pull request and proposed changes adhere to our [contribution guidelines](../../CONTRIBUTING.md) and [code of conduct](../../CODE_OF_CONDUCT.md)
- [ ] Check that tests prove the feature works and covers both happy and unhappy paths
- [ ] Check that there aren't other open Pull Requests for the same update/change

### CODE REVIEW

**Review by danielguillan — APPROVED** [review]
Thanks for fixing this!

File: packages/react/src/IDE/IDE.tsx [hunk 1]
```diff
@@ -129,17 +129,25 @@ type IDEChatProps = {
...
-  highlighter?: 'hljs' // add additional highlighters as needed
+  highlighter?: 'hljs'
+}
+
+type IDEChatMessageAssistant = {
+  role: 'assistant'
+  handle: string
+  message: string
+  codeSnippet?: string
+  highlighter?: 'hljs'
```
Comment (rezrah) [inline packages/react/src/IDE/IDE.tsx:146]: Nit... this seems a little duplicative. Took me a second to spot the difference . Worth creating an intersection with a shared message type for `handle`, `message`,  `codeSnippet` and `highlighter`?

File: packages/react/src/IDE/IDE.tsx [hunk 2]
```diff
@@ -129,17 +129,25 @@ type IDEChatProps = {
...
 }
 
+export type IDEChatMessage = IDEChatMessageUser | IDEChatMessageAssistant
```
Comment (rezrah) [inline packages/react/src/IDE/IDE.tsx:149]: We'll need a changeset for this update as it's a material change to the library. Although, it doesn't need to say too much. 

Maybe something like: Updated `IDEChatMessage` type declaration to differentiate between assistant and user messages. WDYT?

**Review by rezrah — APPROVED** [review]
Thanks @joshfarrant. [Left a comment about a missing changeset](https://github.com/primer/brand/pull/822#discussion_r1848129208), but otherwise LGTM. Will pre-emptively ✅ so you can merge in when ready.

File: packages/react/src/IDE/IDE.tsx (hunk 2 above)
Comment (joshfarrant) [reply packages/react/src/IDE/IDE.tsx]: Good shout, added 👍

File: packages/react/src/IDE/IDE.tsx (hunk 1 above)
Comment (joshfarrant) [reply packages/react/src/IDE/IDE.tsx]: Works for me, updated 🙂

**Comment by joshfarrant** [comment]
Thanks @rezrah 🙌

Just made those changes, will merge once the checks have finished :shipit:

**Pull request merged**